spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -209,9 +211,17 @@ public boolean isUsingStreamTokens() {
 				return Flux.from(instances).doOnNext(t -> writes.add(createUpdateWrite(t)));
 			}
 
-			Flux<List<T>> inputs = Flux.from(instances).bufferTimeout(this.writeBufferSize, this.writeBufferTimeout);
-			return ObservableReactiveUtil.streamingBidirectionalCall(
-					this::openWriteStream, inputs, this::buildWriteRequest);
+			return Flux.from(instances).bufferTimeout(this.writeBufferSize, this.writeBufferTimeout)
```

Comment:
Can we de-dup this and the block in `deleteDocumentsByName()` by wrapping `builder.addWrites(createUpdateWrite(e))` into a lambda?